src/System.Transactions/src/System/Transactions/CommittableTransaction.cs

```diff
@@ -36,18 +35,20 @@ internal CommittableTransaction(IsolationLevel isoLevel, TimeSpan timeout) : bas
             // fill in the traceIdentifier field here.
             _internalTransaction._cloneCount = 1;
             _cloneId = 1;
-            if (DiagnosticTrace.Information)
+            TransactionsEtwProvider etwLog = TransactionsEtwProvider.Log;
+            if (etwLog.IsEnabled())
             {
-                TransactionCreatedTraceRecord.Trace(SR.TraceSourceLtm, TransactionTraceId);
+                etwLog.TransactionCreated(this, "CommittableTransaction");
```

Comment:
Do you know which value to use for "thisOrContextObject" when "this" is not available, such as in static method or static class?